crates/astria-sequencer/src/app.rs

```diff
@@ -291,6 +292,7 @@ impl App {
                 self.executed_proposal_hash = process_proposal.hash;
                 return Ok(());
             }
+            metrics::gauge!(metrics_init::PROCESS_PROPOSAL_SKIPPED_PROPOSAL).increment(1);
```

Comment:
this should be equal to the number of `prepare_proposal` calls, maybe making a metric for `prepare_proposal` would be clearer?